examples/aegis_classifier_example.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
I was wondering if we should move `aegis_classifier_example.py`, `domain_classifier_example.py`, `quality_classifier_example.py`, and `fineweb_edu_classifier_example.py` (when https://github.com/NVIDIA/NeMo-Curator/pull/168 is merged) into their own subdirectory? It could be called something like `huggingface_classifier_examples` or similar.

And then the scripts themselves could be renamed `aegis_example.py`, etc.